Index: app/src/main/java/mx/itesm/jmggm/atizapan/view/MainActivity.kt
===================================================================
diff --git a/app/src/main/java/mx/itesm/jmggm/atizapan/view/MainActivity.kt b/app/src/main/java/mx/itesm/jmggm/atizapan/view/MainActivity.kt
deleted file mode 100644
--- a/app/src/main/java/mx/itesm/jmggm/atizapan/view/MainActivity.kt	(revision 3bebdc61122c58697a228ecc46fbed2531f41cca)
+++ /dev/null	(revision 3bebdc61122c58697a228ecc46fbed2531f41cca)
@@ -1,98 +0,0 @@
-package mx.itesm.jmggm.atizapan.view
-
-import android.content.ContentValues.TAG
-import android.content.Intent
-import android.os.Bundle
-import android.util.Log
-import androidx.appcompat.app.AppCompatActivity
-import com.google.android.gms.auth.api.signin.GoogleSignIn
-import com.google.android.gms.auth.api.signin.GoogleSignInClient
-import com.google.android.gms.auth.api.signin.GoogleSignInOptions
-import com.google.android.gms.common.api.ApiException
-import com.google.firebase.auth.FirebaseAuth
-import com.google.firebase.auth.FirebaseUser
-import com.google.firebase.auth.GoogleAuthProvider
-import com.google.firebase.auth.ktx.auth
-import com.google.firebase.ktx.Firebase
-import mx.itesm.jmggm.atizapan.BottomMenu
-import mx.itesm.jmggm.atizapan.databinding.ActivityMainBinding
-
-class MainActivity : AppCompatActivity() {
-
-    private lateinit var googleSignInClient: GoogleSignInClient
-    private lateinit var auth: FirebaseAuth
-
-    private lateinit var binding: ActivityMainBinding
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        binding = ActivityMainBinding.inflate(layoutInflater)
-        setContentView(binding.root)
-
-        auth = Firebase.auth
-
-        // Configure Google Sign In
-        val gso = GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
-            .requestIdToken("342072291208-nkant9phdi26hqflnu18so9p6i56arvr.apps.googleusercontent.com")
-            .requestEmail()
-            .build()
-
-        googleSignInClient = GoogleSignIn.getClient(this, gso)
-
-        binding.btnSignInGoogle.setOnClickListener {
-            signIn()
-        }
-    }
-
-    private fun signIn() {
-        val signInIntent = googleSignInClient.signInIntent
-        startActivityForResult(signInIntent, RC_SIGN_IN)
-    }
-
-    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
-        super.onActivityResult(requestCode, resultCode, data)
-
-        // Result returned from launching the Intent from GoogleSignInApi.getSignInIntent(...);
-        if (requestCode == RC_SIGN_IN) {
-            val task = GoogleSignIn.getSignedInAccountFromIntent(data)
-            try {
-                // Google Sign In was successful, authenticate with Firebase
-                val account = task.getResult(ApiException::class.java)!!
-                Log.d(TAG, "firebaseAuthWithGoogle:" + account.id)
-                firebaseAuthWithGoogle(account.idToken!!)
-            } catch (e: ApiException) {
-                // Google Sign In failed, update UI appropriately
-                Log.w(TAG, "Google sign in failed", e)
-            }
-        }
-    }
-
-    private fun firebaseAuthWithGoogle(idToken: String) {
-        val credential = GoogleAuthProvider.getCredential(idToken, null)
-        auth.signInWithCredential(credential)
-            .addOnCompleteListener(this) { task ->
-                if (task.isSuccessful) {
-                    // Sign in success, update UI with the signed-in user's information
-                    Log.d(TAG, "signInWithCredential:success")
-                    val user = auth.currentUser
-                    updateUI(user)
-                } else {
-                    // If sign in fails, display a message to the user.
-                    Log.w(TAG, "signInWithCredential:failure", task.exception)
-                    updateUI(null)
-                }
-            }
-    }
-
-    private fun updateUI(user: FirebaseUser?) {
-        if (user != null) {
-            val intent = Intent(applicationContext, BottomMenu::class.java)
-            intent.putExtra(EXTRA_NAME, user.displayName)
-            startActivity(intent)
-        }
-    }
-
-    companion object {
-        const val RC_SIGN_IN = 1001
-        const val EXTRA_NAME = "EXTRA_NAME"
-    }
-}
\ No newline at end of file
Index: app/src/main/res/layout/activity_google_sign.xml
===================================================================
diff --git a/app/src/main/res/layout/activity_google_sign.xml b/app/src/main/res/layout/activity_google_sign.xml
deleted file mode 100644
--- a/app/src/main/res/layout/activity_google_sign.xml	(revision 3bebdc61122c58697a228ecc46fbed2531f41cca)
+++ /dev/null	(revision 3bebdc61122c58697a228ecc46fbed2531f41cca)
@@ -1,28 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".view.GoogleSignInActivity">
-
-    <TextView
-        android:id="@+id/text_display_name"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        android:layout_marginTop="24dp"/>
-
-    <Button
-        android:id="@+id/logout"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="16dp"
-        android:text="Cerrar Sesión"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/text_display_name" />
-
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
Index: app/src/main/java/mx/itesm/jmggm/atizapan/view/GoogleSignInActivity.kt
===================================================================
diff --git a/app/src/main/java/mx/itesm/jmggm/atizapan/view/GoogleSignInActivity.kt b/app/src/main/java/mx/itesm/jmggm/atizapan/view/GoogleSignInActivity.kt
deleted file mode 100644
--- a/app/src/main/java/mx/itesm/jmggm/atizapan/view/GoogleSignInActivity.kt	(revision 3bebdc61122c58697a228ecc46fbed2531f41cca)
+++ /dev/null	(revision 3bebdc61122c58697a228ecc46fbed2531f41cca)
@@ -1,28 +0,0 @@
-package mx.itesm.jmggm.atizapan.view
-
-import android.content.Intent
-import androidx.appcompat.app.AppCompatActivity
-import android.os.Bundle
-import com.google.firebase.auth.ktx.auth
-import com.google.firebase.ktx.Firebase
-import mx.itesm.jmggm.atizapan.BottomMenu
-import mx.itesm.jmggm.atizapan.databinding.ActivityGoogleSignBinding
-import mx.itesm.jmggm.atizapan.view.MainActivity.Companion.EXTRA_NAME
-
-class GoogleSignInActivity : AppCompatActivity() {
-    private lateinit var binding: ActivityGoogleSignBinding
-
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        binding = ActivityGoogleSignBinding.inflate(layoutInflater)
-        setContentView(binding.root)
-
-        binding.textDisplayName.text = intent.getStringExtra(EXTRA_NAME)
-        binding.logout.setOnClickListener {
-            Firebase.auth.signOut()
-
-            val intent = Intent(applicationContext, MainActivity::class.java)
-            startActivity(intent)
-        }
-    }
-}
\ No newline at end of file
Index: app/src/main/java/mx/itesm/jmggm/atizapan/view/SplashActivity.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package mx.itesm.jmggm.atizapan\r\n\r\nimport android.content.Intent\r\nimport androidx.appcompat.app.AppCompatActivity\r\nimport android.os.Bundle\r\nimport android.os.Handler\r\nimport mx.itesm.jmggm.atizapan.view.MainActivity\r\n\r\nclass SplashActivity : AppCompatActivity() {\r\n    lateinit var handler: Handler\r\n    override fun onCreate(savedInstanceState: Bundle?) {\r\n        super.onCreate(savedInstanceState)\r\n        setContentView(R.layout.splash_screen)\r\n\r\n\r\n        handler = Handler()\r\n\r\n        handler.postDelayed({\r\n            val intent = Intent(this, MainActivity::class.java)\r\n            startActivity(intent)\r\n            finish()\r\n        },3000)\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/mx/itesm/jmggm/atizapan/view/SplashActivity.kt b/app/src/main/java/mx/itesm/jmggm/atizapan/view/SplashActivity.kt
--- a/app/src/main/java/mx/itesm/jmggm/atizapan/view/SplashActivity.kt	(revision 3bebdc61122c58697a228ecc46fbed2531f41cca)
+++ b/app/src/main/java/mx/itesm/jmggm/atizapan/view/SplashActivity.kt	(date 1665700101443)
@@ -4,7 +4,6 @@
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.os.Handler
-import mx.itesm.jmggm.atizapan.view.MainActivity
 
 class SplashActivity : AppCompatActivity() {
     lateinit var handler: Handler
@@ -16,7 +15,7 @@
         handler = Handler()
 
         handler.postDelayed({
-            val intent = Intent(this, MainActivity::class.java)
+            val intent = Intent(this, BottomMenu::class.java)
             startActivity(intent)
             finish()
         },3000)
